src/iOS.Essentials/BasicViewExtensions/UIButtonExtensions.cs

```diff
@@ -55,4 +55,12 @@ public static class UIButtonExtensions
         button.Menu = menu;
         return button;
     }
+
+    [SupportedOSPlatform("ios14.0")]
+    public static T MakeMenuButton<T>(this T button, Func<UIMenu> menu) where T : UIButton

```

Comment:
Func<UIMenu> is a reference to a method.
Func<UIMenu> **getMenu** sounds more correct because it gives the idea of not actually having a menu, but rather a way of getting it.

Reply:
I agree with that